main.go

```diff
@@ -140,7 +140,7 @@ func ensureFastlaneVersionAndCreateCmdSlice(forceVersion, gemfilePth string) ([]
 			if exist, err := pathutil.IsPathExists(gemfileLockPth); err != nil {
```

Comment:
This check will fail, even if successful as gemfileLockPth is empty. Will need to call gems.GemFileLockPth again.

Reply:
Updated.